app/assets/schemas/archive-format.json

```diff
@@ -0,0 +1,203 @@
+          "type": "string"
+        },
+        "email": {
+          "id": "https://the-federation.info/specs/backup-restore#archive_format/user/email",
+          "type": "string"
+        },
+        "language": {
+          "id": "https://the-federation.info/specs/backup-restore#archive_format/user/language",
+          "type": "string"
+        },
+        "username": {
+          "id": "https://the-federation.info/specs/backup-restore#archive_format/user/username",
+          "type": "string"
+        },
+        "private_key": {
+          "id": "https://the-federation.info/specs/backup-restore#archive_format/user/private_key",
+          "type": "string"
+        },
+        "profile": {
+          "id": "https://the-federation.info/specs/backup-restore#archive_format/user/profile",
+          "type": "object",
+          "properties": {
+            "first_name": {
+              "id": "https://the-federation.info/specs/backup-restore#archive_format/user/profile/first_name",
+              "type": "string"
+            },
+            "last_name": {
+              "id": "https://the-federation.info/specs/backup-restore#archive_format/user/profile/last_name",
+              "type": "string"
+            },
+            "gender": {
+              "id": "https://the-federation.info/specs/backup-restore#archive_format/user/profile/gender",
+              "type": "string"
+            },
+            "bio": {
+              "id": "https://the-federation.info/specs/backup-restore#archive_format/user/profile/bio",
+              "type": "string"
+            },
+            "location": {
+              "id": "https://the-federation.info/specs/backup-restore#archive_format/user/profile/location",
+              "type": "string"
+            },
+            "image_url": {
+              "id": "https://the-federation.info/specs/backup-restore#archive_format/user/profile/image_url",
+              "type": "string"
+            },
+            "thefederation_uid": {
+              "id": "https://the-federation.info/specs/backup-restore#archive_format/user/profile/thefederation_uid",
+              "type": "string"
+            }
+          }
+        },
+        "contacts": {
+          "id": "https://the-federation.info/specs/backup-restore#archive_format/user/contacts",
+          "type": "array",
+          "items": {
+            "id": "https://the-federation.info/specs/backup-restore#archive_format/user/contacts/185",
+            "type": "object",
+            "properties": {
+              "sharing": {
+                "id": "https://the-federation.info/specs/backup-restore#archive_format/user/contacts/185/sharing",
+                "type": "boolean"
+              },
+              "following": {
+                "id": "https://the-federation.info/specs/backup-restore#archive_format/user/contacts/185/receiving",
+                "type": "boolean"
+              },
+              "person_thefederation_uid": {
```

Comment:
Maybe not for discussion here, but I find that name rather weird. I'd prefer something more neutral like `accountIdentifier` or similar. What do you think?